conseil-common/src/main/scala/tech/cryptonomic/conseil/common/cache/MetadataCaching.scala

```diff
@@ -16,22 +16,22 @@ object MetadataCaching {
 
   /** Class representing key in cache */
   sealed trait CacheKey {
-    val value: String
+    def get: String
   }
-  case class EntityCacheKey(platform: String, network: String) extends CacheKey {
-    val value: String = s"$platform.$network"
+  case class EntitiesCacheKey(platform: String, network: String) extends CacheKey {
+    def get: String = s"$platform.$network"
```

Comment:
why `def` here?

Reply:
Mostly, to do not cache yet another field, which can be easily re-computed, but can be `val` if you want :)